code/modules/awaymissions/mission_code/snowdin.dm

```diff
@@ -595,7 +595,7 @@
 	death = FALSE
 	faction = ROLE_SYNDICATE
 	outfit = /datum/outfit/snowsyndie
-	flavour_text = {"You are a syndicate operative recently awoken from cyrostatis in an underground outpost. Monitor Nanotrasen communications and record information. All intruders should be
+	flavour_text = {"You are a syndicate operative recently awoken from cryostatis in an underground outpost. Monitor Nanotrasen communications and record information. All intruders should be
 	disposed of swirfly to assure no gathered information is stolen or lost. Try not to wander too far from the outpost as the caves can be a deadly place even for a trained operative such as yourself."}
```

Comment:
While correcting spelling you might as well fix this one to how its supposed to be, 

> All intruders should be disposed of by swirly to assure no gathered information is stolen.

Reply:
lmfao